packages/elastic_agent/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.19.0"
+  changes:
+    - description: Add output metrics
```

```suggestion
    - description: Add more detailed output metrics
```

nit: current description is a bit vague